apache/couchdb
# Add couch_mrview_debug:view_signature/2

## Overview

This PR introduces a function to display view signature which corresponds to a given ddoc. 
The function is intended for using in remsh while debugging view upgrade issues.

## Testing recommendations

1. start couch `dev/run --admin=adm:pass`
2. start `remsh` session `dev/remsh`
3. call a function for a database which has a ddoc `couch_mrview_debug:view_signature(<<"db321">>, <<"ddoc321">>).`
4. make sure the help message for a function makes sense `couch_mrview_debug:help(view_signature).`
5. make sure the function is listed in an output from `couch_mrview_debug:help().`

## Related Issues or Pull Requests

N/A

## Checklist
